cmake/dependencies/AMReX.cmake

```diff
@@ -1,4 +1,10 @@
 macro(find_amrex)
+    # if pyAMReX is external, AMReX must be as well
+    if(DEFINED WarpX_pyamrex_internal AND NOT WarpX_pyamrex_internal)
```

Comment:
Is this a typo? Shouldn't this be
```
if(DEFINED WarpX_amrex_internal AND NOT WarpX_pyamrex_internal)
```

Reply:
No, that is intentional:
`if(NOT WarpX_pyamrex_internal)` is TRUE if the variable is truly false-valued or undefined. Thus, I first check if its defined, then check if the value is negative.